airflow/providers/amazon/aws/transfers/redshift_to_s3.py

```diff
@@ -101,6 +107,14 @@ def __init__(  # pylint: disable=too-many-arguments
         self.include_header = include_header
         self.table_as_file_name = table_as_file_name
 
+        self._select_query = None
+        if select_query:
+            self._select_query = select_query
+        elif self.schema and self.table:
+            self._select_query = f"SELECT * FROM {self.schema}.{self.table}"
+        else:
+            raise ValueError('Please provide both `schema` and `table` params or `select_query` to fetch the data.')
```

Comment:
```suggestion
            raise ValueError(
                'Please provide both `schema` and `table` params or `select_query` to fetch the data.'
            )
```

Reply:
This should fix failing Pylint error: https://github.com/apache/airflow/pull/14177/checks?check_run_id=1873280195#step:7:137

Reply:
Would recommend installing pre-commit to identify this before running `git commit`:

More details: https://github.com/apache/airflow/blob/master/STATIC_CODE_CHECKS.rst

Reply:
Thank you for pointers @kaxil !